src/tls/TLSClient.fst

```diff
@@ -0,0 +1,298 @@
+module TLSClient
```

Comment:
These files existed before this PR and they will be merged, so it's not worth it.